client/src/components/Overview/sampleData.js

```diff
@@ -0,0 +1,40 @@
+let sampleData = {
```

Comment:
Do we need this file in production at all?

Reply:
I have deleted the sample data